### ISSUE TITLE
SPDX correction

### ISSUE THREAD

**Issue by schuylermartin45** [conda-incubator/conda-recipe-manager#45]
From `r-highr-feedstock`:
```
Error:   × Parsing: failed to parse SPDX license: unknown term
  │ See <https://spdx.org/licenses> for the list of valid licenses.
  │ Use 'LicenseRef-<MyLicense>' if you are using a custom license.
    ╭─[75:12]
 74 │ about:
 75 │   license: GPL-3
    ·            ──┬──
    ·              ╰── here
 76 │   summary:
    ╰────
```

Ideas:
- We should throw a warning when a license is not recognized in the conversion phase
- We _could_ potentially auto-correct the issue(?)

**Comment by schuylermartin45** [comment]
As of writing, 169 Anaconda recipes in the integration tests have this failure.

**Comment by schuylermartin45** [comment]
Some number of these appear to be slightly mis-labeled.

We might be able to get pretty far by just performing some simple spelling-like patches.

For example, a ton of R packages use `GPL-3` while the SPDX-recognized string is `GPL-3.0`.

**Comment by schuylermartin45** [comment]
A first pass of this was implemented in #55
